src/lib/expandApplyAtRules.js

```diff
@@ -115,13 +122,24 @@ function expandApplyAtRules(context) {
 
           let rules = applyClassCache.get(applyCandidate)
 
+          candidates.push([applyCandidate, important, rules])
+        }
+      }
+
+      for (const [parent, candidates] of perParentApplies) {
+        let siblings = []
+
+        for (let candidate of candidates) {
+          let applyCandidate = candidate[0]
+          let important = candidate[1]
+          let rules = candidate[2]
```

Comment:
Little nitpick, I think it might be cleaner if we write it as:

```suggestion
        for (let [applyCandidate, important, rules] of candidates) {
```

I double checked and as far as I can tell, we don't need the actual `candidate` so inlining it should be fine.

Reply:
Yep, totally I was surgically moving stuff around and didn't think to collapse this. Good call!